src/EnergyPlus/ChilledCeilingPanelSimple.cc

```
@@ -1015,29 +1015,62 @@ namespace CoolingPanelSimple {
 		
 		RegisterPlantCompDesignFlow( CoolingPanel( CoolingPanelNum ).WaterInletNode, CoolingPanel( CoolingPanelNum ).WaterVolFlowRateMax );
 
+		bool SizeUAErrorFlag = false;
+		SizeCoolingPanelUA( CoolingPanelNum, SizeUAErrorFlag );
+		if ( SizeUAErrorFlag ) ShowFatalError( "SizeCoolingPanelUA: Program terminated for previous conditions." );
+
+	}
+
+	void
+	SizeCoolingPanelUA(
+					 int const CoolingPanelNum,
+					 bool & SizeUAError
```

Could we change this function so rather than returning the error status via the reference it returns true on success and false on failure? It's a more "C++" way of doing this.

---

@jasondegraw Any objections to lumping this comment together with some other C++ modifications that have been requested?  That is already the subject for Followup for new cooling panel ZoneHVAC:CoolingPanel:RadiantConvective:Water (GitHub Issue #5823).

---

@RKStrand That sounds good to me.

---

@RKStrand I think maybe I was unclear on what I was suggesting here, sorry if that was the case. What I was suggesting here was to change the function signature to `bool SizeCoolingPanelUA(int const CoolingPanelNum)` rather than use a reference.